src/x11/x11-platform.c

```diff
@@ -709,6 +709,21 @@ static void eplX11CleanupDisplay(EplDisplay *pdpy)
     }
 }
 
+static EGLBoolean CheckX11Extension(xcb_connection_t *conn, const char *name)
+{
+    xcb_generic_error_t *error = NULL;
+    xcb_query_extension_reply_t *reply = xcb_query_extension_reply(conn,
+            xcb_query_extension(conn, strlen(name), name), &error);
+    EGLBoolean ret = EGL_FALSE;
+    if (reply != NULL)
+    {
+        ret = (reply->present != 0);
+        free(reply);
+    }
+    free(error);
```

Comment:
Is there a way we could log an error here? Might help us with triage to know why egl-x11 is failing to find extension support.